packages/gatsby-plugin-theme-ui/test/provider.js

```diff
@@ -23,6 +23,7 @@ test('renders with theme context', () => {
   const root = render(wrapRootElement({ element: <Consumer /> }, {}))
   expect(context.theme).toEqual({
     colors: {},
+    rawColors: {},
     styles: {
       pre: {},
     },
```

Comment:
Not sure I understand the logic of this test completely.

I adapted it because it was failing, but I'll let you guys see if that's actually what we want.

Reply:
:thinking: It tests... just if the context provider is used by wrapRootElement? Okay, let's leave it.